### PR TITLE
chore(apps/interface): refactor LightmodeIcon

### PR DESCRIPTION
**Pull request by math-marcellino**
## Scope
List of affected projects:

- apps/interface

## Description
We need to refactor the `LightmodeIcon` component.

Return `Icon` directly instead of using `return` keyword.

Update this:

```
const LightmodeIcon = (props: IconProps) => {
	return <Icon />
}
```

to this:

```
const LightmodeIcon = (props: IconProps) => (
	<Icon />
)
```

## Action items
Action items for this pull request:

- [x] Refactor the return function

## Checklist
You should check this before requesting for review:

- [x] Branch name use the the following format RIS-{NUMBER}
- [x] Pull request title is "chore(apps/interface): refactor LightmodeIcon"
- [x] Make sure CSS styling is same as the spec (padding, border etc)
- [x] Make sure new files are 100% covered in [Risedle Code Coverage](https://coverage.risedle.com)

### CODE REVIEW

File: apps/interface/components/Icons/Lightmode.tsx [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+            fill="currentColor"
+        />
+    </Icon>
+);
 export default LightmodeIcon;
```
Comment (pyk) [inline apps/interface/components/Icons/Lightmode.tsx:13]: @math-marcellino minor. please add whitespace above this line

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/components/Icons/Lightmode.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/components/Icons/Lightmode.tsx:13]: gotcha, sorry I missed it sir

Comment (pyk) [reply apps/interface/components/Icons/Lightmode.tsx:13]: @math-marcellino all good. please check other pull request yes. thank you

Comment (math-marcellino) [reply apps/interface/components/Icons/Lightmode.tsx:14]: Gotcha @pyk , fixed for this PR 👍

**Review by pyk — APPROVED** [review]

**Pull request merged**